<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZHPlayerBaseUIClass</class>
 <widget class="QMainWindow" name="ZHPlayerBaseUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZHPlayer</string>
  </property>
  <property name="windowOpacity">
   <double>0.500000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="m_pBtnLastMusic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnNextMusic">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnPlay">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QSlider" name="m_pSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="m_pLabelMusicName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnVolue">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>35</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>音量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnChangeType">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>35</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnOpen">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>35</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZHPlayerBaseUI.qrc"/>
 </resources>
 <connections/>
</ui>
